Source/WebCore/svg/SVGToOTFFontConversion.cpp

```diff
@@ -166,10 +166,10 @@ class SVGToOTFFontConverter : public CanMakeWeakPtr<SVGToOTFFontConverter> {
 
     void append32BitCode(ASCIILiteral code)
     {
-        m_result.append(code[0]);
-        m_result.append(code[1]);
-        m_result.append(code[2]);
-        m_result.append(code[3]);
+        m_result.append(code[0UZ]);
```

Comment:
Do you like upper case `0UZ` rather than `0uz`?
https://en.cppreference.com/w/cpp/language/integer_literal
Personally, I prefer lower case. I don't know which does WebKit  prefer.

Reply:
I definitely do not care. :) Happy to change to lowercase before landing.